### PR TITLE
dx-643 Two-phase payments alert

### PR DESCRIPTION
**Pull request by arebra**
Added an alert regarding two-phase payments. Added it in the Capture.md where it is possible, and in the "good to know" section elsewhere. Perhaps a bit harsh having a warning in the introduction?

### CODE REVIEW

File: _includes/alert-two-phase-payments.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% include alert.html type="warning" icon="warning" header="Two-phase payments"
```
Comment (payex-dev) [inline _includes/alert-two-phase-payments.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6097051)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6097051&bid=19960937)

File: _includes/alert-two-phase-payments.md (hunk 1 above)
Comment (asbjornu) [reply _includes/alert-two-phase-payments.md:1]: Please add a newline at the end of the file.

**Review by asbjornu — APPROVED** [review]
I think this is great!

**Review by marinafrydenb — APPROVED** [review]

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6097051&bid=19960937)

File: _includes/alert-two-phase-payments.md (hunk 1 above)
Comment (payex-dev) [inline _includes/alert-two-phase-payments.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6097051)

File: _includes/alert-two-phase-payments.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{% include alert.html type="warning" icon="warning" header="Two-phase payments" 
+body="When dealing in physical goods using two-phase payment instruments, all 
+goods _must_ be shipped **before** you capture the authorized amount. This is 
+regulated in part by the Swedish Distance and Doorstep Sales Act, required in 
+our terms and (for card payments) required by Visa and Mastercard." %}
```
Comment (asbjornu) [inline _includes/alert-two-phase-payments.md:5]:
```suggestion
our terms and (for card payments) required by Visa and Mastercard." %}

```

**Review by asbjornu — CHANGES_REQUESTED** [review]
Seems like `_includes/alert-two-phase-payments.md` is still missing a newline at the end of the file.

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6097051&bid=19960937)

File: _includes/alert-two-phase-payments.md (hunk 1 above)
Comment (payex-dev) [inline _includes/alert-two-phase-payments.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6097051)

**Review by asbjornu — APPROVED** [review]

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6097051&bid=19960937)

File: _includes/alert-two-phase-payments.md (hunk 1 above)
Comment (payex-dev) [inline _includes/alert-two-phase-payments.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6097051)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6097051&bid=19960937)

File: _includes/alert-two-phase-payments.md (hunk 1 above)
Comment (payex-dev) [inline _includes/alert-two-phase-payments.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6097051)

**Pull request merged**